### PR TITLE
Adds skeleton pages for footer items

### PR DESCRIPTION
**Pull request by rossjones**
This PR adds footer items for:

* A privacy policy page
* A cookies page
* A terms and conditions page

They are currently mostly empty (except the cookies page) and we will
need to find content before these are at all usable.

### CODE REVIEW

File: app/views/view_data/pages/cookies.html.erb [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+  <div class="grid-row">
+    <div class="column-full">
+      <h1 class="heading-large">
+        Cookies
+      </h1>
+    </div>
+  </div>
+
+  <br/>
```
Comment (Rosa-Fox) [inline app/views/view_data/pages/cookies.html.erb]: Could get rid of this <br/> as it isn't semantically a line break in a sentence... more for spacing?

File: spec/controllers/view_data/page_controller_spec.rb [hunk 2]
```diff
@@ -8,5 +8,20 @@
...
+
+    it 'has a privacy policy page' do
+      get :privacy
+      expect(controller).to have_rendered('view_data/pages/privacy')
+    end
+
+    it 'has a terms and conditions page' do
+      get :terms
+      expect(controller).to have_rendered('view_data/pages/terms')
+    end
```
Comment (Rosa-Fox) [inline spec/controllers/view_data/page_controller_spec.rb]: Looks good. We could potentially add a feature spec to check the footer links work and contain the correct content, but not sure if that would be overkill?

File: app/views/view_data/pages/cookies.html.erb (hunk 1 above)
Comment (rossjones) [reply app/views/view_data/pages/cookies.html.erb]: More for 'forgot to remove it' really.

Comment (Rosa-Fox) [reply app/views/view_data/pages/cookies.html.erb]: 👍

**Pull request merged**